house/views.py

```diff
@@ -156,9 +156,11 @@ def search(request):
         house_list = list()
         for each in geo_search_res:
             house = House.objects.get(pk=each["house_id"])
-            if house.number_of_beds >= num_of_beds and house.date_begin <= date_begin and house.date_end >= date_end:
+            if house.number_of_beds >= num_of_beds and house.date_begin <= date_begin and house.date_end >= date_end and \
+                    check_if_this_time_can_book(house, date_begin, date_end):
                 house_info = house.dict_it()
                 house_info["longitude"], house_info["latitude"] = each["location"]["coordinates"]
+                house_info["active"] = True
                 house_list.append(house_info)
         return JsonResponse({
```

Comment:
Return all rooms, including those unavailable ones, but set `active` to `False` for them, so it's clearer when demoing.